src/app/core/views/Preferences/tabs/Account/Language.tsx

```diff
@@ -10,9 +10,9 @@ import dayjs from 'dayjs';
 import ItemsDropdown from './components/ItemsDropdown';
 import MenuItem from './components/MenuItem';
 
-const localStorageLanguage = localStorageService.get('language');
+const localStorageLanguage = localStorageService.get('i18nextLng');
 
-const languages = ['en', 'es', 'fr', 'it', 'cn', 'ru', 'de'];
+const languages = ['en', 'es', 'fr', 'it', 'zh', 'ru', 'de'];
```

Comment:
also need to change `types.ts` in i18n folder

Reply:
Done